{{cookiecutter.project_slug}}/docs/index.md

```diff
@@ -12,6 +12,9 @@
 
 ## Features
 
+- This package uses [pydocstyle](http://www.pydocstyle.org/en/stable/)
```

Comment:
you will need to check first if the user selected pydocstyle.
do it in a new PR.